test/manifest.ts

```diff
@@ -283,6 +286,77 @@ describe('Manifest', () => {
       ]);
     });
 
+    it('allows root module to be published, via special "." path', async function () {
+      const manifest = JSON.stringify({
+        'node/pkg1': '3.2.1',
+        'node/pkg2': '1.2.3',
+        '.': '2.0.0',
+      });
+      const config = JSON.stringify({
+        packages: {
+          'node/pkg1': {},
+          'node/pkg2': {},
+          '.': {},
+        },
+      });
+      const commits = [
+        buildMockCommit('feat(@node/pkg1)!: major new feature', [
+          'node/pkg1/src/foo.ts',
+        ]),
+        buildMockCommit('feat(@node/pkg2)!: major new feature', [
+          'node/pkg2/src/bar.ts',
+        ]),
+      ];
```

Comment:
the `googleapis/google-api-nodejs-client` use case ("aggregate" package) makes the most sense to me but I wonder if other folks might have a setup where by non-submodule changes could be important to the `'.'` package? I'm not saying we need to extend existing functionality, but it would be good to understand what the current behavior would be if say we added a commit to this list like

```typescript
        buildMockCommit('fix(root): root only change', [
          'src/foo.ts',
        ])
```

in this particular case the version bumping would be shadowed by the `!` commits on submodules but I'd expect at least an entry in the root changelog?

Reply:
Added a test case 👍